src/resources/views/crud/fields/relationship.blade.php

```diff
@@ -34,6 +34,18 @@
             $field['type'] = 'relationship_select';
         }
     }
+    }elseif(in_array($field['relation_type'], ['MorphToMany', 'BelongsToMany', 'MorphMany', 'HasMany'])) {
```

Comment:
This will be easier to read as part of the `switch` statement like I suggested above.

Reply:
Ok .. joining this makes sense ..